### PR TITLE
[WIP] Internal docs

### PR DESCRIPTION
**Pull request by zipy124**
## What does this PR do?
Adds our own documentation, to make development easier.

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?
- [ ] Are docs written for this PR? (if applicable)

## 🚨 Is this a breaking change for API clients?
no

## :squirrel: Deploy notes
nothing to note

## Anything else
Instructions in the readme

### CODE REVIEW

**Comment by zipy124** [comment]
All imports are done, we're ready to roll-out though it would be nice to properly docstring everything!

File: backend/uclapi/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'uclapi.settings')
+
+import django
```
[the remark this answers is not in the document]
Comment (zipy124) [reply backend/uclapi/source/conf.py]: This is unfortunately needed :(

**Comment by zipy124** [comment]
example of formatted docstring implemented: 
image

File: backend/uclapi/timetable/utils.py [hunk 2]
```diff
@@ -13,6 +13,20 @@
 
 
 def get_location_coordinates(siteid, roomid):
+    """
+    Given a site and room, returns the co-ordinates of this location.
+
+    :param siteid:
+    :type siteid: string
```
Comment (Feverfew) [inline backend/uclapi/timetable/utils.py]:
```suggestion
    :type siteid: str
```

File: backend/uclapi/timetable/utils.py [hunk 3]
```diff
@@ -13,6 +13,20 @@
 
 
 def get_location_coordinates(siteid, roomid):
+    """
+    Given a site and room, returns the co-ordinates of this location.
+
+    :param siteid:
+    :type siteid: string
+    :param roomid:
+    :type roomid: string
```
Comment (Feverfew) [inline backend/uclapi/timetable/utils.py]:
```suggestion
    :type roomid: str
```

File: backend/uclapi/timetable/utils.py [hunk 4]
```diff
@@ -13,6 +13,20 @@
...
+    :param siteid:
+    :type siteid: string
+    :param roomid:
+    :type roomid: string
+
+    :raise Location.DoesNotExist: If the site can not be found
+
+    :returns: latitude and longitude, or None,None in failure to
+              raise exception
+    :rtype: tuple (string,string)
```
Comment (Feverfew) [inline backend/uclapi/timetable/utils.py]:
```suggestion
    :rtype: tuple (str,str)
```

**Pull request merged**